contents/blog/making-something-people-want.md

```diff
@@ -0,0 +1,149 @@
+
+It was instantly popular.
+
+It taught us that consolidating multiple tools into the platform made us much more valuable to users. It taught us that we could pull that off. Those things meant that we could compete in cloud and wouldn't be reliant on self hosting for the whole business.
+
+Code won.
+
+## Throw away bad ideas
+
+The most perfect pre-product-market-fit startup, is a machine that repeatedly (i) learns (ii) ships (iii) goes back to step (i).
+
+If you ship something, and learn that it's not popular (which will happen _most_ of the time), then you have a decision - do I _iterate_ this thing, or do I _start over_?
+
+Startup lore is that most people will bias heavily towards iteration when they should just start over. Dalton, one our partners at YC, [has a great talk on this topic](https://www.ycombinator.com/library/6p-all-about-pivoting). It's understandable - you don't want to lose face by changing ideas.
+
+Having an ego is expensive. For us, it was better value to throw the majority of our ideas away.
+
+## Who does what
+
+This must be simple for the greatest speed.
+
+I found users, built the website and docs, and the odd wireframe.
+
+Tim (my cofounder and our CTO) dealt with existing users by shipping stuff.
+
+## Ship for real life people
+
+As soon as we had decided to build PostHog, our next phase was to see if anyone would use it.
+
+Before launching on HN, we focused on this loop:
+
+1. Find someone who is interested in what we working on
+
+This was the only time we had to do outreach at PostHog. Once we had product market fit, this need disappeared.
+
+I knew we were targeting developers, so with that in mind "I need a developer to implement and hopefully give feedback on this". We had no focus on money yet.
+
+I went through (i) friends and family that sprung to mind (ii) literally my phonebook (iii) 1st connections on LinkedIn (iv) every developer I'd ever worked with via LinkedIn (v) every developer who'd worked at every place I'd ever worked.
+
+I decided WhatsApp / SMS > Email > LinkedIn, depending on if I had each person's contact details. With ideas that didn't work out, it was harder work to get people to try it!
+
+All messages were totally cutomized to the person. Nothing automated. I was very clear I was just hoping that we could get them as an early, free user - because we were preparing the product before a big launch.
+
+Within a couple of days I had a few teams who were up for trying it.
+
+We spent a further $2K on ads on Twitter to see if we could get a little self serve usage up and running - our main focus here was to (i) check nothing was broken, in case it went viral when we launched and (ii) get the repo to 100 stars. I'd stress we had a hard 4 week deadline we'd set ourselves, so we were spending money to avoid spending more time to let the above happen naturally. This step wasn't necessary, so please don't copy if you've no capital.
```

Comment:
```suggestion
We spent a further $2K on ads on Twitter to see if we could get a little self serve usage up and running - our main focus here was to (i) check nothing was broken, in case it went viral when we launched and (ii) get the repo to 100 stars. I'd stress we had a hard 4 week deadline we'd set ourselves, so we were spending money to avoid spending more time to let the above happen naturally. This step wasn't necessary, so please don't copy it if you have no capital.
```